app/javascript/components/public_health/PatientsTable.js

```diff
@@ -358,17 +425,24 @@ class PatientsTable extends React.Component {
     );
   };
 
-  updateAssignedUsers(jurisdiction_id, scope, workflow, tab) {
-    if (tab !== 'transferred_out') {
+  /**
+   * Method to update the datalist of assigned users & the Patients Table
+   * @param {Object} query - updated query fo patients table
+   */
+  updateAssignedUsers = query => {
+    if (query.tab !== 'transferred_out') {
       axios
         .post('/jurisdictions/assigned_users', {
-          query: { jurisdiction: jurisdiction_id, scope, workflow, tab },
+          query: { jurisdiction: query.jurisdiction, scope: query.scope, workflow: query.workflow, tab: query.tab },
         })
         .then(response => {
           this.setState({ assigned_users: response.data.assigned_users });
+          this.updateTable({ ...this.state.query, ...query });
```

Comment:
While working, I noticed that `updateTable()` was called after each time `updateAssignedUsers()` was called. So I removed all those individual `updateTable()` calls and just included it as a part of this method. If anyone notices any possible issues with this change, let me know. (I will note that I also included a call to `updateTable()` a couple lines down to also cover the case when the user is on the Transferred Out line list)

Reply:
Hmm, I’m not sure I like having this as an after-effect in this specific method because updating the datalist of assigned users doesn’t intrinsically require we also update the table, it just happens the be the case most of the time that we update the table after. I would prefer these be separated out since they are two separate actions. I see what you mean by us almost always calling `updateTable` after calling `updateAssignedUsers` because when the assigned users need to update, the table also does in most cases. Given that, I’d be okay with there being a new method that extracts out these two separate actions called like `updateAssignedUsersAndTable` that calls both methods. Thoughts?

Reply:
I agree with what Kylie said in that although both actions tend to be called in the same places, they are separate actions. Another thing with this change in implementation is now these requests are not being fired simultaneously because `this.updateTable` is only called after the assigned_users request is completed

Reply:
Kylie and I discussed this in more detail last week so I'm following up here regarding some digging I did since then. When I was initially working on this enhancement, the `updateAssignedUsers()` call was not properly updating the assigned users drop down list. This was the original reason I made this update. But just now, after reverting this particular aspect of the PR, it appears that other adjustments in this PR resolved the issue and the assigned user drop down now correctly populates even when `updateAssignedUsers()` is call directly after in it's own function call. As a result, I will revert this particular aspect of my PR to go along with the comments in this thread. This adjustment will be included in my next commit.